app/src/main/java/gholzrib/arctouchchallenge/core/utils/LoadingDialog.java

```diff
@@ -0,0 +1,40 @@
+package gholzrib.arctouchchallenge.core.utils;
+
+/**
+ * Created by Gunther Ribak on 08/01/2015.
+ * For more information contact me
+ * through guntherhr@gmail.com
+ */
+
+import android.app.ProgressDialog;
+import android.content.Context;
+
+public class LoadingDialog extends ProgressDialog {
+
+    private static final String MESSAGE_LOADING = "Loading";
```

Comment:
I tried to make this class the most independent as possible, so I decided to create the String locally.